vagrant-spk

```diff
@@ -191,6 +191,72 @@ def call_vagrant_command(sandstorm_dir, *command_args):
     print("Calling {} in {}".format(" ".join(["'{}'".format(arg) for arg in command]), sandstorm_dir))
     return subprocess.check_call(command, cwd=sandstorm_dir)
 
+def ensure_working_vboxsf_in_base_box(sandstorm_dir):
+    # If the .sandstorm/Vagrantfile refers to debian/jessie64, and does not pin it to a particular
+    # version, and the user has version 8.2.2 available on their system, then we abort the
```

Comment:
I'd like to start by just checking for known-bad versions; hopefully 8.2.3 can revert this, though I doubt it, but let's hope for that for now IMHO.